### PR TITLE
Update migration page and small fixes

### PR DESCRIPTION
**Pull request by thedriftofwords**
- Bulk migration added to UI
- Make forwarder info part of the main procedure
- Minor changes in Getting Started

### CODE REVIEW

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx [hunk 1]
```diff
@@ -47,14 +45,33 @@ Upkeeps that are successfully migrated will show the following [transaction logs
 
 <ClickToZoom src="/images/automation/v2-migration/migration-txn.png" />
 
+### Migrating multiple upkeeps
+
+If you have multiple upkeeps to migrate, you can do so in the Chainlink Automation App.
+
+1. Click the link displays on your upkeeps dashboard. This link only displays if you have one or more upkeeps to migrate:
```
Comment (dwightjl) [inline src/content/chainlink-automation/guides/migrate-to-v2.mdx:52]:
```suggestion
1. Click the link displays on your upkeeps dashboard. This link displays only if you have one or more upkeeps to migrate:
```

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx [hunk 2]
```diff
@@ -47,14 +45,33 @@ Upkeeps that are successfully migrated will show the following [transaction logs
...
+
+   <ClickToZoom src="/images/automation/v2-migration/1-bulk-migration-ui.png" />
+
+1. Follow the prompts to approve and confirm the transactions to migrate your upkeep(s).
```
Comment (dwightjl) [inline src/content/chainlink-automation/guides/migrate-to-v2.mdx:56]:
```suggestion
1. Follow the prompts to approve and confirm the transactions to migrate your upkeeps.
```

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx [hunk 3]
```diff
@@ -69,6 +86,13 @@ To migrate one or more upkeeps using the scanner:
 
 1. Execute the `migrateUpkeeps` function and confirm the transaction. When this transaction is confirmed, your upkeeps will be migrated to the latest registry, and each upkeep will have a [unique forwarder](#unique-forwarder) address.
 
+1. Update your contract to use the forwarder address by following the instructions in the [Forwarder](/chainlink-automation/guides/forwarder) page.
```
Comment (dwightjl) [inline src/content/chainlink-automation/guides/migrate-to-v2.mdx:89]:
```suggestion
1. Update your contract to use the forwarder address by following the instructions on the [Forwarder](/chainlink-automation/guides/forwarder) page.
```

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx [hunk 4]
```diff
@@ -69,6 +86,13 @@ To migrate one or more upkeeps using the scanner:
...
+
+When the migration is complete:
```
Comment (dwightjl) [inline src/content/chainlink-automation/guides/migrate-to-v2.mdx:91]:
```suggestion
After the migration is complete:
```

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx [hunk 5]
```diff
@@ -69,6 +86,13 @@ To migrate one or more upkeeps using the scanner:
...
+
+- Your balance is transferred to the new registry automatically.
+- Read the [migration checklist](#migration-checklist) to understand further updates you may need to make in your contracts.
```
Comment (dwightjl) [inline src/content/chainlink-automation/guides/migrate-to-v2.mdx:94]:
```suggestion
- Read the [migration checklist](#migration-checklist) to understand further updates you might need to make in your contracts.
```

File: src/content/chainlink-automation/overview/getting-started.mdx [hunk 6]
```diff
@@ -92,9 +92,9 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
 
       1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Sepolia** in the top dropdown menu.
 
-      1. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
+      1. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://faucets.chain.link/arbitrum-sepolia).
```
Comment (dwightjl) [inline src/content/chainlink-automation/overview/getting-started.mdx:95]:
```suggestion
      1. Connect your cryptocurrency wallet to the app if you haven't done so already. You might also need to fetch LINK for the Arbitrum Sepolia testnet from [faucets.chain.link](https://faucets.chain.link/arbitrum-sepolia).
```

File: src/content/chainlink-automation/overview/getting-started.mdx [hunk 7]
```diff
@@ -65,7 +65,7 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
...
```
Comment (dwightjl) [inline src/content/chainlink-automation/overview/getting-started.mdx:68]:
```suggestion
      1. Connect your cryptocurrency wallet to the app if you haven't done so already. You might also need to fetch LINK for the Arbitrum Sepolia testnet from [faucets.chain.link](https://faucets.chain.link/arbitrum-sepolia).
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx [hunk 8]
```diff
@@ -47,14 +45,33 @@ Upkeeps that are successfully migrated will show the following [transaction logs
...
+
+1. Click the link displays on your upkeeps dashboard. This link only displays if you have one or more upkeeps to migrate:
+
+   <ClickToZoom src="/images/automation/v2-migration/1-bulk-migration-ui.png" />
+
+1. Follow the prompts to approve and confirm the transactions to migrate your upkeep(s).
+
+   <ClickToZoom src="/images/automation/v2-migration/approve-migration.png" />
+
+### Updating permissions
```
Comment (dwightjl) [inline src/content/chainlink-automation/guides/migrate-to-v2.mdx:60]: Is this required for bulk migration? If yes, add a step to point users to this section instructing them to update their permissions. Also, does this apply to the "Migrating using the Chainlink Automation App" section?

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: src/content/chainlink-automation/guides/migrate-to-v2.mdx (hunk 8 above)
Comment (thedriftofwords) [reply src/content/chainlink-automation/guides/migrate-to-v2.mdx]: It potentially applies to any type of migration, so I restructured it to reflect that. Thanks!

**Review by dwightjl — APPROVED** [review]

**Pull request merged**